spec/models/hotel_spec.rb

```diff
@@ -3,5 +3,8 @@
 RSpec.describe Hotel, type: :model do
   describe "relationships" do
     it { should have_many :rooms }
+    it { should have_many(:guests).through(:rooms) }
```

Comment:
This is the correct shoulda-matcher for this association but the association is not represented in the actual model.